pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -239,15 +237,6 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxWindowsConfig(pod *v1.Pod)
 		SecurityContext: &runtimeapi.WindowsSandboxSecurityContext{},
 	}
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.WindowsHostNetwork) {
-		wc.SecurityContext.NamespaceOptions = &runtimeapi.WindowsNamespaceOption{}
-		if kubecontainer.IsHostNetworkPod(pod) {
-			wc.SecurityContext.NamespaceOptions.Network = runtimeapi.NamespaceMode_NODE
-		} else {
-			wc.SecurityContext.NamespaceOptions.Network = runtimeapi.NamespaceMode_POD
-		}
-	}
-
```

Comment:
The logic were removed after marking the FG as Deprecated. Is this a general pattern? 

IIRC, we should remove the logical once we removed the FG(or locked the FG to true/false).